proj/airframe.conf

```diff
@@ -1,28 +1,15 @@
-// https://github.com/wvlet/airframe.git#49158a1832af354c411a6feeb2241d0c059be885
-
-// frozen (April 2021) at known-green commit before it started
-// consistently timing out during testing; I've set a reminder to
-// investigate
+// https://github.com/wvlet/airframe.git#master
 
 vars.proj.airframe: ${vars.base} {
   name: "airframe"
-  uri: "https://github.com/wvlet/airframe.git#49158a1832af354c411a6feeb2241d0c059be885"
+  uri: "https://github.com/wvlet/airframe.git#8f186064b63913dcf104f7c321f50e2d9208a9c1"
 
   check-missing: false  // ignore missing scripted-plugin
   extra.projects: ["communityBuild"]  // no Scala.js plz
   extra.exclude: [
-    "airspecJVM"  // we use a binary, because of https://github.com/wvlet/airframe/pull/1380
+    "airspecJVM", "airspecRefJVM"  // we use binaries, because of https://github.com/wvlet/airframe/pull/1380
```

Comment:
Note: Now that airspec supports Scala 3.0.0, we no longer need to use binary. We should be able to remove "airspecRefJVM" from this exclude list.